packages/schematics/angular/migrations/update-8/differential-loading.ts

```diff
@@ -171,10 +128,37 @@ function getCompilerOptionsAstObject(host: Tree, tsConfigPath: string): JsonAstO
     return;
   }
 
+  const configExtends = findPropertyInAstObject(tsCfgAst, 'extends');
+  const isExtendedConfig = configExtends && configExtends.kind === 'string';
```

Comment:
@clydin asked this check to be stricter:

> Can we add some safety checks to the extends checks for the tsconfig?  If it extends from more than one or if the depth of extension is more than one, it should probably bail.
> a config that’s heavily customized will probably break if the migration starts modifying options

Reply:
Let’s take this on slack, bailing out increases the changes of leaving broken projects as this migration is mandatory for other migrations such as for the lazy loading import syntax change.

Reply:
sounds right to me - users can always look at the diff to understand what we did